hack/update-codegen.sh

```diff
@@ -74,59 +77,63 @@ go run ./pkg/generator
 new_report="$(mktemp -t "$(basename "$0").api_violations.XXXXXX")"
 
 echo "Generating openapi schema"
-go run k8s.io/code-generator/cmd/openapi-gen \
-  -O zz_generated.openapi \
+go run k8s.io/kube-openapi/cmd/openapi-gen \
+  --output-file zz_generated.openapi.go \
   --report-filename "${new_report}" \
-  --output-package "sigs.k8s.io/gateway-api/pkg/generated/openapi" \
-  --input-dirs "${GATEWAY_INPUT_DIRS}" \
-  --input-dirs "k8s.io/apimachinery/pkg/apis/meta/v1" \
-  --input-dirs "k8s.io/apimachinery/pkg/runtime" \
-  --input-dirs "k8s.io/apimachinery/pkg/version" \
-  ${COMMON_FLAGS}
+  --output-dir "pkg/generated/openapi" \
+  --output-pkg "sigs.k8s.io/gateway-api/pkg/generated/openapi" \
+  ${COMMON_FLAGS} \
+  $GATEWAY_INPUT_DIRS_SPACE \
+  k8s.io/apimachinery/pkg/apis/meta/v1 \
+  k8s.io/apimachinery/pkg/runtime \
+  k8s.io/apimachinery/pkg/version
```

Comment:
@dprotaso can you help get this across the line, although this is not erroring out, it's also not working for me

Reply:
🤦 figured it out - I hadn't included input dirs correctly in the section below.